contents/blog/Building-the-future-of-game-analytics-pureskill.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2021-21-09
+title: Building the future of game analytics - An interview with PureSkill.gg CTO, Evan Sosenko
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: joe-martin
+featuredImage: ../images/blog/posthog-game-analytics-pureskill.png
+featuredImageType: full
+---
+_Welcome to another episode of PostHog's Community Conversations, where we chat to our contributors and users. This time, we speak to Evan Sosenko, Co-founder and CTO of PureSkill.gg._
+
+<iframe width="560" height="315" src="https://youtu.be/x1jxCJb9zII" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+[PureSkill.gg](https://pureskill.gg/) is an automated coach for Counter-Strike: Global Offensive (CS:GO), one of the most popular multiplayer games in the world. Rather than merely showing you statistics about how you play, PureSkill.gg identifies mistakes you make and teaches you how to overcome them.
+
+As a Co-founder and CTO, Evan Sosenko is responsible for all technical aspects of PureSkill.gg’s automated coaching system and spends most of his time using product insights to build features which improve the experience for users. In addition to collecting quantitative insights from PostHog, Evan also engages the community directly to get qualitative feedback. 
+
+Sometimes that qualitative feedback can lead to surprising results. In our conversation with Evan he told us about his favourite experience at PureSkill.gg and how one user on the community Discord went from reporting bugs to becoming an angel investor in the company.
+
+To people who play a lot of games, the idea of automated games coaching will probably sound a little strange, however Evan is quick to point out how rapidly the eSports and competitive gaming industry is growing.  
+
+“_My favourite example is Dota because I’m a Dota fan,_” said Evan. “_They (game developer Valve Software) host [a tournament every year with the top Dota players](https://www.dota2.com/international/overview). Originally it was just in Seattle, but now it’s truly international. It started with a $1 million dollar prize pool ten years ago, but now it’s almost a $40 million dollar prize pool. So, I feel like we’re really just getting started._”
```

Comment:
Do the readers need to know that Dota is a game. Could be confused as an acronym?

Reply:
It technically is an acronym (it stands for Defense of the Ancients) but has been nominalised such that it is no longer used as one. 

I personally think Dota is well known enough and should be obvious from the context.